main.py

```diff
@@ -65,12 +69,17 @@ def upload_and_notify(files: List, drive_conn: drive.DriveAPI, slack_conn: slack
   for file in files:
     try:
       # Get url from upload function.
-      file_url = drive_conn.upload_file(file['file'], file['name'])
+      file_url = drive_conn.upload_file(file['file'], file['name'], file['folder_id'])
+
+      # The formatted date/time string
+      fall_back = f"{file['date']} UTC"
 
       # Only post message if the upload worked.
-      message = f'The recording of _{file["meeting"]}_ on _{file["date"]} UTC_ is <{file_url}| ' \
-                f'now available>.'
-      slack_conn.post_message(message)
+      message = f'The recording of _{file["meeting"]}_ on '
+      message += "_<!date^" + str(file['unix']) + "^{date} at {time}|" + fall_back + ">_"
+      message += f' is <{file_url}| now available>.'
```

Comment:
Thanks I improved it. It still is a bit a mix and match because of the f-strings and + signs because Slack formatting using `{date}` and {`time}` but it no longer uses the `+=` format.